ambry-rest/src/main/java/com.github.ambry.rest/NettyMetrics.java

```diff
@@ -237,5 +247,24 @@ public NettyMetrics(MetricRegistry metricRegistry) {
     // NettyServer
     nettyServerShutdownTimeInMs = metricRegistry.histogram(MetricRegistry.name(NettyServer.class, "ShutdownTimeInMs"));
     nettyServerStartTimeInMs = metricRegistry.histogram(MetricRegistry.name(NettyServer.class, "StartTimeInMs"));
+    // ConnectionStatsHandler
+    connectionsConnectedCount =
```

Comment:
Discussed this offline - just need to change the `NettyServer` into `ConnectionStatsHandler`